pkg/ansible/proxy/requestfactory/requestinfo.go

```diff
@@ -125,7 +125,8 @@ type RequestInfoFactory struct {
 // /api/{version}
 // /api
 // /healthz
-// /
+
+//nolint:gocyclo
```

Comment:
This is not enabled in the CI.
To test the same lint checks locally which are made in the CI you can run `./hack/tests/check-lint.sh ci`